tests/jvm/src/test/scala/scalaz/std/java/EnumTest.scala

```diff
@@ -33,11 +33,11 @@ object EnumTest extends SpecLite {
   }
 
   "pred" in {
-    TimeUnit.values.sliding(2) forall { case Array(tu1, tu2) => tu1 === tu2.pred }
+    TimeUnit.values.sliding(2).map(_.toList) forall { case List(tu1, tu2) => tu1 === tu2.pred }
```

Comment:
does `case Seq(` not work?

Reply:
`scala.Array#sliding` return 
- `Iterator[Array[Int]]` in Scala 2.12
- `Iterator[scala.collection.mutable.WrappedArray[Int]]` in Scala 2.13.0-M4-pre-20d3c21

```
Welcome to Scala 2.12.4 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_172).
Type in expressions for evaluation. Or try :help.

scala> Array(1, 2, 3, 4).sliding(2).map{ case Seq(a, b) => }
<console>:12: error: scrutinee is incompatible with pattern type;
 found   : Seq[A]
 required: Array[Int]
       Array(1, 2, 3, 4).sliding(2).map{ case Seq(a, b) => }
                                                 ^
```

Reply:
Sad that you can't patmat a WrappedArray with Seq..

Reply:
You can still match it as a `scala.collection.Seq` but not as a `scala.Seq` anymore because it's a mutable collection.

Reply:
Oh yeah, cool 😄